app/assets/stylesheets/application.scss.erb

```diff
@@ -2028,17 +2055,17 @@ and it won't be important on password protected instances */
     left: 0;
     width: 100%;
     height: 100%;
-    position: fixed;
+    position: absolute;
```

Comment:
This is a big change, one of a few which allows scrolling in lightboxes. I haven't tested this very thoroughly on other lightboxes